### DISCUSSION TITLE
Conda is not installed. Exiting...

### DISCUSSION THREAD

**Discussion by aminlv** [oobabooga/text-generation-webui#2846] [Q&A]
need help on installing on ubuntu 22.10

no matter when i use sudo or not right after installing miniconda it crash and exit
i even manually installed conda still same issue

i appreciate your help in advance

./start_linux.sh 
Downloading Miniconda from https://repo.anaconda.com/miniconda/Miniconda3-py310_23.1.0-1-Linux-x86_64.sh to /home/user/oobabooga_linux/installer_files/miniconda_installer.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 70.9M  100 70.9M    0     0   689k      0  0:01:45  0:01:45 --:--:--  779k
PREFIX=/home/user/oobabooga_linux/installer_files/conda
Unpacking payload ...
                                                                                                                      
Installing base environment...

Downloading and Extracting Packages

Downloading and Extracting Packages

Preparing transaction: done
Executing transaction: done
installation finished.
Miniconda version:
conda 23.1.0
Conda is not installed. Exiting...

**Comment by aminlv** [comment]
well no one replied 
if anyone had similar problem change webui.py 

```
check_env():

    #conda_exist = run_cmd("conda", environment=True, capture_output=True).returncode == 0
    conda_exist = True
```

**Reply by Seantourage** [reply]
Another solution if you don't want to edit the webui.py which could cause git pull upgrade issues due to edited code or adding the directory to your path which could mess up other conda projects, just do the following:

Copy _conda.exe from installer_files\conda to the root directory and rename it to conda.exe
